scripts/cc-cli/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
`scripts/cc-cli` needs to be added to the `javascript-*` jobs in ci.yml, see for example https://github.com/gluwa/creditcoin/blob/dev/.github/workflows/ci.yml#L384